packages/@ember/-internals/glimmer/tests/integration/components/input-angle-test.js

```diff
@@ -621,30 +627,34 @@ moduleFor(
             },
           },
         });
-      }, /Passing actions to components as strings \(like `({{input key-press="foo"}}|<Input @key-press="foo" \/>)`\) is deprecated\./);
+      }, /(Passing actions to components as strings \(like `({{input key-press="foo"}}|<Input @key-press="foo" \/>)`\) is deprecated\.|Passing the `@key-press` argument to <Input> is deprecated\.)/);
```

Comment:
It's probably better to do this in this case:

```js
maybeExpectDeprecation(
  EMBER_MODERNIZED_BUILT_IN_COMPONENTS,
  () => {
    expectDeprecation(
      () => this.render(...),
      /Passing actions to components as strings \(like `({{input key-press="foo"}}|<Input @key-press="foo" \/>)`\) is deprecated\./
    );
  },
  /Passing the `@key-press` argument to <Input> is deprecated\./
);
```

Reply:
I tried this and it seems to result in getting the deprecation at the wrong time: 

![Screen Shot 2021-02-10 at 11 22 20 AM](https://user-images.githubusercontent.com/13040530/107560504-49a6d680-6b92-11eb-904e-ba0257508c25.png)

Full test for reference: 

```['@test [DEPRECATED] sends an action with `<Input @key-press="foo" />` is pressed'](assert) {
      assert.expect(4);
      maybeExpectDeprecation(
        EMBER_MODERNIZED_BUILT_IN_COMPONENTS,
        () => {
          expectDeprecation(
            () => {
              this.render(`<Input @value={{this.value}} @key-press='foo' />`, {
                value: 'initial',
                actions: {
                  foo(value, event) {
                    assert.ok(true, 'action was triggered');
                    if (jQueryDisabled) {
                      assert.notOk(event.originalEvent, 'event is not a jQuery.Event');
                    } else {
                      assert.ok(event instanceof jQuery.Event, 'jQuery event was passed');
                    }
                  },
                },
              });
            },
            /Passing actions to components as strings \(like `({{input key-press="foo"}}|<Input @key-press="foo" \/>)`\) is deprecated\./
          );
        },
        /Passing the `@key-press` argument to <Input> is deprecated\./
      );```

Reply:
Oh hm, I guess expectDeprecation doesn’t support being nested, that’s quite unfortunate but maybe what you has is the best we could do without refactoring that infrastructure. Sorry about that. @rwjblue can you confirm?